tasks/gulp/__tests__/after-build-package.test.js

```diff
@@ -107,17 +107,19 @@ describe('package/', () => {
       return readFile(filePath, 'utf8')
         .then((data) => {
           var parsedData = JSON.parse(data)
+
           // We expect the component JSON to contain "name", "type", "required", "description"
-          expect(parsedData).toEqual(
-            expect.arrayContaining([
+          expect(Array.isArray(parsedData)).toBeTruthy()
```

Comment:
I _think_ this could be…

```suggestion
          expect(parsedData).toBeInstanceOf(Array)
```